<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsServiceConf</class>
 <widget class="QDialog" name="QgsServiceConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>.pg_service.conf Editor</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PostgreSQL Services bearbeiten:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxServices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hostname:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Datenbank:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Benutzername:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Passwort:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditHost"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPort"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditDatabase"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditUsername"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Service:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditService"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Neuen Service erstellen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ausgewählten Service löschen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Serviceänderungen speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Hilfe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
